test/test_replaygain.py

```diff
@@ -132,6 +141,20 @@ def test_cli_saves_album_gain_to_file(self):
 class ReplayGainGstCliTest(ReplayGainCliTestBase, unittest.TestCase):
     backend = u'gstreamer'
 
+    def setUp(self):
+        try:
+            # Check if required plugins can be loaded by instantiating a
+            # GStreamerBackend (via its .__init__).
+            config['replaygain']['targetlevel'] = 89
+            GStreamerBackend(config['replaygain'], None)
+        except FatalReplayGainError as e:
+            # Skip the test if plugins could not be loaded.
+            if str(e) == "Failed to load required GStreamer plugins":
```

Comment:
I have added an explicit check against the exception message, as `FatalReplayGainError` is used for several purposes. Might seem a bit redundant, but it is mainly for making the sure that the test is skipped only for the "missing plugins" reason, and it does not hide other errors in the future (as, at the moment, there seems to be no way to reach this line except for that reason).